openmdao/code_review/test_lint_docstrings.py

```diff
@@ -551,9 +551,10 @@ def test_docstrings(self):
                         funcs = []
 
                     for func_name in funcs:
-                        func = getattr(mod, func_name)
-                        self.check_function(dir_name, file_name, func_name,
-                                            func, failures)
+                        if not func_name.startswith('_'):
+                            func = getattr(mod, func_name)
```

Comment:
Are we sure we want to do this (team?)  The way we use underscore methods, it is nice to have our style of docstring, and if we remove them from the testing, then we will have to rely on catching them in the reviews.

Reply:
Note that this only applies to module level '_'* functions.  Are you suggesting that even the most private function buried in a module should be forced to have our full, verbose docstring?  That seems like overkill IMHO.  For example, should the following _make_interp_spline function have a full docstring?
```
try:
    from scipy.interpolate._bsplines import make_interp_spline as _make_interp_spline
except ImportError:
    def _make_interp_spline(*args, **kwargs):
        msg = "'MetaModelStructuredComp' requires scipy>=0.19, but the currently" \
              " installed version is %s." % scipy_version
        raise RuntimeError(msg)
```

Reply:
I don't have a problem with putting a docstring there, but we could support a "lint ignore" directive syntax for this as well.

Reply:
i agree with @naylor-b  here. We don't need to force people to lint internal module level private function